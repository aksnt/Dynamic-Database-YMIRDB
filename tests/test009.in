SET a 1 2 3
SET b 4 5 a
PLUCK a 2
PLUCK b 4
PICK b 4
PLUCK b 3
PICK b 3
BYE
