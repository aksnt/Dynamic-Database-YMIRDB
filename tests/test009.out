> ok

> ok

> 2

> index out of range

> index out of range

> a

> index out of range

> bye
